lib/evmone/eof.hpp

```diff
@@ -0,0 +1,29 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2020 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+#pragma once
+
+#include <evmc/utils.h>
+#include <cstddef>
+#include <cstdint>
+#include <string>
+
+namespace evmone
+{
+using bytes_view = std::basic_string_view<uint8_t>;
+
+struct EOF1Header
+{
+    size_t code_size = 0;
+    size_t data_size = 0;
+
+    [[nodiscard]] EVMC_EXPORT size_t code_begin() const noexcept;
```

Comment:
@chfast Does this look ok to mark functions in this file `EVMC_EXPORT` in order to call them from unit tests?

Reply:
Yes, although you may reconsider you really need to test it if it's trivial function.